### PR TITLE
main.html: Add fork me github corner

### PR DESCRIPTION
**Pull request by rohanrk**
This html adds a github corner to the site.
https://github.com/tholman/github-corners for reference

Closes https://github.com/coala/gci-leaders/issues/26

### CODE REVIEW

File: templates/main.html [hunk 1]
```diff
@@ -21,6 +21,17 @@
       </a>
       <h1 data-i18n="page-title">Google Code-in 2017 Current Leaders</h1>
     </div>
+    <div class="ribbon">
+      <a href="https://your-url" class="github-corner" aria-label="View source on Github">
```
Comment (adtac) [inline templates/main.html]: fix the href

File: templates/main.html [hunk 2]
```diff
@@ -21,6 +21,17 @@
...
+        <svg width="80" height="80" viewBox="0 0 250 250" style="fill:#151513; color:#fff; position: absolute; top: 0; border: 0; right: 0;" aria-hidden="true">
```
Comment (adtac) [inline templates/main.html]: Nobody likes *that* guy, but this code is licensed under MIT. MIT is a subset of (A)GPL, so it's fine importing that code as long as we don't change the copyright. Maybe just mention that this code is from https://github.com/tholman/github-corners/blob/master/license.md, which is MIT licensed and preserve the copyright notice?

Not sure what's the right approach here. Maybe I'm just being unnecessarily pedantic. CC @jayvdb

**Review by adtac — DISMISSED** [review]
the href needs to changed in any case

**Comment by rohanrk** [comment]
That would be a problem. Should I mention the copyright and display a link to the corner repo in the footer of the site?

**Review by jayvdb — CHANGES_REQUESTED** [review]
Please add a comment in the code acknowledging the author / repo https://github.com/tholman/github-corners ; or maybe we could load the SVG & CSS from that repo using rawgit.com

File: templates/main.html [hunk 3]
```diff
@@ -21,6 +21,17 @@
       </a>
       <h1 data-i18n="page-title">Google Code-in 2017 Current Leaders</h1>
     </div>
+    <div class="ribbon">
```
Comment (varunzxzx) [inline templates/main.html]: Add the copyright notice `Copyright (c) 2016 Tim Holman` as comment just above the code. So, that we don't infringe the rights.

File: templates/main.html [hunk 4]
```diff
@@ -21,6 +21,17 @@
...
+          <path d="M115.0,115.0 C114.9,115.1 118.7,116.5 119.8,115.4 L133.7,101.6 C136.9,99.2 139.9,98.4 142.2,98.6 C133.8,88.0 127.5,74.4 143.8,58.0 C148.5,53.4 154.0,51.2 159.7,51.0 C160.3,49.4 163.2,43.6 171.4,40.1 C171.4,40.1 176.1,42.5 178.8,56.2 C183.1,58.6 187.2,61.8 190.9,65.4 C194.5,69.0 197.7,73.2 200.1,77.6 C213.8,80.2 216.3,84.9 216.3,84.9 C212.7,93.1 206.9,96.0 205.4,96.6 C205.1,102.4 203.0,107.8 198.3,112.5 C181.9,128.9 168.3,122.5 157.7,114.1 C157.9,116.9 156.7,120.9 152.7,124.9 L141.0,136.5 C139.8,137.7 141.6,141.9 141.8,141.8 Z" fill="currentColor" class="octo-body">
+          </path>
+        </svg>
+      </a>
+      <style>.github-corner:hover .octo-arm{animation:octocat-wave 560ms ease-in-out}@keyframes octocat-wave{0%,100%{transform:rotate(0)}20%,60%{transform:rotate(-25deg)}40%,80%{transform:rotate(10deg)}}@media (max-width:500px){.github-corner:hover .octo-arm{animation:none}.github-corner .octo-arm{animation:octocat-wave 560ms ease-in-out}}</style>
```
Comment (varunzxzx) [inline templates/main.html]: Move the styling to the head section. It will make sure that the styles will load before the DOM renders.

**Review by varunzxzx — CHANGES_REQUESTED** [review]

**Review by ShivamSinghania — CHANGES_REQUESTED** [review]
In addition to changes requested above, edit your shortlog. Include "fork me ribbon" in the shortlog for more clarity. http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

File: templates/main.html (hunk 4 above)
Comment (blazeu) [reply templates/main.html]: Styling goes in css file

Comment (varunzxzx) [reply templates/main.html]: Yeah, you can do that as well.

**Comment by jayvdb** [comment]
I dont remember why this was blocked.

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Review by jayvdb — APPROVED** [review]

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
ack https://github.com/coala/gci-leaders/pull/147/commits/0cf25331e9911da9ba1696b19716dc5478b5043a

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**